### DISCUSSION TITLE
Help Getting Started

### DISCUSSION THREAD

**Discussion by littonztl** [Weatherlights/Winget-AutoUpdate-Intune#57] [General]
I'm so confused about the steps required to make this work and I really would like to implement this to manage some pesky apps that we continuously churn on to keep updated.

Step 1:
Upload ADMX/ADML into Intune
Which files need to be uploaded?
Winget-AutoUpdate-Intune-main\ADMX? or Winget-AutoUpdate-Intune-main\Sources\Policies\ADMX? or both?

Step 2:
Deploy the Winget-AutoUpdate-aaS client

I used the Microsoft Store deployment for Winget-AutoUpdate-aaS. Does this require that Romanitho.Winget-AutoUpdate is also installed? I'm just a little confused is this tool supposed to be used in replace of or in conjunction with WAU.

Step 3:
Managing Policies

Does it assume all other apps shall be installed if you use the block list?

Thanks in advance!

**Comment by WalterSobchak9515** [comment]
Step 1:  Yes, you need to upload both the ADMX and ADML files.  Go to devices, then configuration inside Intune to upload those files, once you select to import them, you will get this screen:

![image](https://github.com/user-attachments/assets/59b2bb33-56f2-4adb-a748-5083ce6031c6)

Step 2:  Not sure about the MS Store as I can't deploy most MS Store apps in my tenant as it hasn't been updated to the "new" store yet like commercial tenant has.  I used the MSI file supplied by weatherlights to deploy via intune apps.  That has everything you need in it.  Then you just configure the winget policies in Intune.  They will be available to configure after you upload the admx adml files.

I created a new policy and called it Winget auto update, then I go into that policy to change the winget settings.

Step 3:  Not sure what you are asking but we only use this to update applications that are currently installed onto the PC.  You can use a whitelist (what I suggest in an organization) or a blacklist.  Whitelist will only update the applications that you specify.  Blacklist will update everything that winget is able to update, except what you put on the blacklist.
